deltacat/compute/resource_estimation/delta.py

```diff
@@ -42,7 +42,11 @@ def _estimate_resources_required_to_process_delta_using_previous_inflation(
     in_memory_size = (
         delta.meta.content_length * estimate_resources_params.previous_inflation
     )
-    num_rows = int(in_memory_size / estimate_resources_params.average_record_size_bytes)
+    num_rows = 0
+    if estimate_resources_params.average_record_size_bytes is not None:
```

Comment:
Are we ok with returning `num_rows = 0` in this case?

Reply:
It's the current behavior.

Reply:
If `estimate_resources_params.average_record_size_bytes is None`, we return `num_rows = 0` rather than getting an exception?

Reply:
Yes, that behavior is in driver.